Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -50,6 +50,14 @@ class BackForwardTableViewCell: UITableViewCell {
         didSet {
             if let s = site {
                 faviconView.setFavicon(forSite: s, onCompletion: { [weak self] (color, url) in
+                    
+                    // Mozilla Firefox Start Page
+                    let urlString = (url?.isLocal)! ? url?.getQuery()["url"]?.unescape() : url?.absoluteString
```

Comment:
Be careful about using `!` on optionals - especially on a property of an optional value. In these cases, always handle the nil case gracefully by either using a conditional if (`if let x = x`) or a guard (`guard let x = x`).